src/swsssdk/dbconnector.py

```diff
@@ -9,182 +9,324 @@
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
 class SonicDBConfig(object):
+    SONIC_DB_GLOBAL_CONFIG_FILE = "database_global.json"
     SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_dir = "/var/run/redis/sonic-db"
     _sonic_db_config_init = False
     _sonic_db_config = {}
 
+    @staticmethod
+    def load_sonic_global_db_config(global_db_file_path):
+        """
+        Parse and load the global database config json file
+        """
+        if os.path.isfile(global_db_file_path) ==  True:
+            with open(global_db_file_path, "r") as read_file:
+                all_dbs = json.load(read_file)
+                for entry in all_dbs['INCLUDES']:
+                    if 'namespace' not in entry.keys():
+                        ns = ''
+                    else: 
+                        ns = entry['namespace']
+
+                    db_include_file = os.path.join(SonicDBConfig._sonic_db_config_dir, entry['include'])
```

Comment:
>db_include_file = os.path.join(SonicDBConfig._sonic_db_config_dir, entry['include']) [](start = 20, length = 84)

We need to support `include` relative path relative to `global_db_file_path`. Not relative to SONIC_DB_BASE_DIR.